contk/dataloader/language_generation.py

```diff
@@ -201,15 +217,17 @@ def trim_index(self, index):
 				* ignore `<pad>` s at the end of the sentence.
 
 		Arguments:
-				index (list): a list of int
+			index (list): a list of int
 
 		Examples:
-				>>> dataloader.index_to_sen(
-				...		[])
-				>>>
-
-		Todo:
-				* fix the missing example
+			vocab_list = ["<pad>", "<unk>", "<go>", "<eos>", "I", "have",\
+							"been", "to", "Sichuan"]
+			>>> dataloader.trim_index(
+			...		[2, 4, 5, 6, 7, 8, 3, 0, 0])
+			>>> [2, 4, 5, 6, 7, 8]
+			Hint:
+			original sentence: <go> I have been to Sichuan <pad> <pad> <eos> I <eos> <pad>
+			trim index result: <go> I have been to Sichuan
 		'''
```

Comment:
How about writing comment in examples?
```

>>> dataloader.trim_index(
...		[2, 4, 5, 6, 7, 8, 3, 0, 0]) # <go> I have been to Sichuan <eos> <pad> <pad>
>>> [2, 4, 5, 6, 7, 8]  # <go> I have been to Sichuan

```
And also rewrite the other example like this way may be better.